BackEnd/src/main/kotlin/quest/marketstack/tradingapp/model/ClosedTrade.kt

```diff
@@ -0,0 +1,82 @@
+package quest.marketstack.tradingapp.model
+
+import org.springframework.data.annotation.Id
+import org.springframework.data.mongodb.core.mapping.Document
+
+@Document(collection = "\${api.closed-trade.collection}")
+data class ClosedTrade(
+    @Id
+    val id: String? = null,
+    var tradeExecs: MutableCollection<TradeExec> = mutableListOf(),
+    val shortLong: Boolean,
+) {
+    val getProfitLoss: Double get() {
```

Comment:
I had said on discord not to have this logic in here, I take back that statement, you can keep all this logic here